de.metas.swat/de.metas.swat.base/src/main/java/de/metas/invoicecandidate/modelvalidator/C_Order.java

```diff
@@ -128,6 +128,14 @@ private boolean isCheckCreditLimitNeeded(@NonNull final I_C_Order order)
 			return false;
 		}
 
+		final IBPartnerStatsDAO bpartnerStatsDAO = Services.get(IBPartnerStatsDAO.class);
+		final I_C_BPartner partner = InterfaceWrapperHelper.load(order.getC_BPartner_ID(), I_C_BPartner.class);
+		final BPartnerStats stats = bpartnerStatsDAO.getCreateBPartnerStats(partner);
```

Comment:
use the ` IBPartnerStatsDAO.getCreateBPartnerStats(final int bpartnerId)` u just introduced.